src/main/kotlin/ecommerce/auth/interceptor/CheckLoginInterceptor.kt

```diff
@@ -0,0 +1,33 @@
+package ecommerce.auth.interceptor
+
+import ecommerce.auth.infrastructure.AuthorizationExtractor
+import ecommerce.auth.infrastructure.BearerAuthorizationExtractor
+import ecommerce.auth.infrastructure.JwtTokenProvider
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.springframework.web.servlet.HandlerInterceptor
+
+class CheckLoginInterceptor(private val jwtTokenProvider: JwtTokenProvider) : HandlerInterceptor {
+    private val authorizationExtractor: AuthorizationExtractor<String> = BearerAuthorizationExtractor()
+
+    override fun preHandle(
+        request: HttpServletRequest,
+        response: HttpServletResponse,
+        handler: Any,
+    ): Boolean {
+        val token = authorizationExtractor.extract(request)
+        println("CheckLoginInterceptor: token=$token")
+        if (token.isNullOrBlank()) {
```

Comment:
```suggestion
        if (token.isBlank()) {
```

This should be fine, since `token` can't be null.